pipeline/dune

```diff
@@ -0,0 +1 @@
+(dirs :standard \ var)
```

Comment:
Similar to the toplevel dune file, this is necessary as with multiple packages defined in `pipeline/dune-project`, dune checks all directories, including `pipeline/var` which is the ocurrent working directory.